apps/mobile/src/components/GroupWelcomeCheck/GroupWelcomeCheck.js

```diff
@@ -5,25 +5,15 @@ import useCurrentGroup from '@hylo/hooks/useCurrentGroup'
 
 export default function GroupWelcomeCheck () {
   const navigation = useNavigation()
-  const [{ currentUser }] = useCurrentUser()
```

Comment:
Remove useCurrentUser import, and please get your linting working :) This file still imports useCurrentUser though it doesn't utilize it anymore.